pcsx2/DebugTools/DebugInterface.cpp

```diff
@@ -184,6 +184,36 @@ void DebugInterface::resumeCpu()
 		core.Resume();
 }
 
+
+char *DebugInterface::stringFromPointer(u32 p)
+{
+	static char buf[25] = { 0 };
+
+	if (!isValidAddress(p))
+		return NULL;
+
+	try {
+		for (u32 i = 0; i < 25; i++) {
+			char c = read8(p + i);
+			buf[i] = c;
+
+			if (c == 0) {
+				return i > 0 ? buf : NULL;
+			}
+			else if (c < 0x20 || c >= 0x7f) {
+				// non printable character
+				return NULL;
+			}
+		}
+	}
+	catch (Exception::Ps2Generic&) {
+		return NULL;
+	}
+	buf[25] = 0;
+	buf[24] = '~';
```

Comment:
Hum tell me, if I'm wrong. The variable is buf[25] so it range from 0 to 24. buf[25] seem outside.